ihmc-common-walking-control-modules/src/test/java/us/ihmc/commonWalkingControlModules/capturePoint/splitFractionCalculation/SplitFractionToolsTest.java

```diff
@@ -0,0 +1,162 @@
+package us.ihmc.commonWalkingControlModules.capturePoint.splitFractionCalculation;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static us.ihmc.commons.MathTools.clamp;
+
+public class SplitFractionToolsTest
```

Comment:
Can you add a few tests for right at the epsilon bounds? To make sure that the epsilon is enforced. So see what happens if the return is just above the minimum epsilon, right at the minimum epsilon, and just below the minimum epsilon. THat's quite literally the definition of your edge case here.